certified-connectors/RAPID Platform/apiDefinition.swagger.json

```diff
@@ -0,0 +1,645 @@
+										}
+									},
+									"value-path": "Schema"
+								}
+							}
+						}
+					}
+				}
+			}
+		},
+		"/lists/{listName-dynamic}/items/{itemId}": {
+			"get": {
+				"summary": "Get an item",
+				"operationId": "GetItem",
+				"description": "Fetches a specific item from a Table on your RAPID site. This will always return the full item profile of the fetched item.",
+				"parameters": [
+					{
+						"name": "listName-dynamic",
+						"type": "string",
+						"in": "path",
+						"description": "Select the list you want to fetch the item from.",
+						"required": true,
+						"x-ms-url-encoding":"single",
+						"x-ms-summary": "Select List",
+						"x-ms-dynamic-values": {
+							"operationId": "GetLists"
+						}
+					},
+					{
+						"in": "path",
+						"name": "itemId",
+						"required": true,
+						"description": "Numeric ID of the item to get",
+						"x-ms-summary": "Item ID",
+						"x-ms-url-encoding":"single",
+						"type": "integer"
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "The A specific item",
+						"schema": {
+							"type": "object",
+							"x-ms-dynamic-schema": {
+								"operationId": "GetSingleItemSchema",
+								"parameters": {
+									"listName": {
+										"parameter": "listName-dynamic"
+									}
+								},
+								"value-path": "Schema"
+							}
+						}
+					}
+				}
+			},
+			"put": {
+				"summary": "Update an item",
+				"operationId": "UpdateListItem",
+				"description": "Update an item",
+				"x-ms-summary": "Update an item",
+				"tags": [
+					"lists"
+				],
+				"security": [
+					{
+						"oauth2_auth": []
+					}
+				],
+				"parameters": [
+					{
+						"name": "listName-dynamic",
+						"type": "string",
+						"in": "path",
+						"description": "Select the list you want to update",
+						"required": true,
+						"x-ms-summary": "Select List",
+						"x-ms-url-encoding":"single",
+						"x-ms-dynamic-values": {
+							"operationId": "GetLists"
+						}
+					},
+					{
+						"in": "path",
+						"name": "itemId",
+						"required": true,
+						"description": "Numeric ID of the item to update",
+						"type": "integer",
+						"x-ms-url-encoding":"single",
+						"x-ms-summary": "Item Id"
+					},
+					{
+						"x-ms-summary": "Item Profile",
+						"name": "dynamicListSchema",
+						"in": "body",
+						"description": "Dynamic schema of selected list",
+						"schema": {
+							"type": "object",
+							"x-ms-dynamic-schema": {
+								"operationId": "GetSingleItemSchema",
+								"parameters": {
+									"listName": {
+										"parameter": "listName-dynamic"
+									}
+								},
+								"value-path": "Schema"
+							}
+						}
+					}
+				],
+				"responses": {
+					"204": {
+						"description": "The item was updated successfully"
+					}
+				}
+			}
+		},
+		"/lists/{listName-dynamic}/items/{itemId}/inherited-links": {
+			"get": {
+				"summary": "Get a inherit links for a given item",
```

Comment:
@MattFranklin-Rapid It should be `"Get an inherit link for given item"` or `"Get inherit links for given item"`

Reply:
Added missing word list to inherited-links summaries. On inspection we are not sure if there should be both a summary and x-ms-summary key so have updated both for the moment.